src/app/data-planes/views/DataPlanePoliciesView.vue

```diff
@@ -163,17 +165,35 @@
                 </template>
               </div>
             </template>
+            <RouterView
+              v-if="route.params.policyPath && route.params.policy"
```

Comment:
When these update the entire view re-renders, which means that if you have any accordions open, they close again. If you rebase here you should be able to use `route.child()` instead i.e. "does this route have a child to show" if it does then show the summary view.

Once you've done that you should be able to remove lines 8 and 9 above. Which means the route has less dependencies and therefore re-renders less, and crucially, doesn't re-render when the `policy` or `policyPath` updates.

Reply:
Ok, this is great 🙌 I removed the route params and use `route.child()` instead. However this wasn't enough. At least locally it still rerendered the whole route and closed the accordions. I was able to achieve the expected behaviour by relocating and duplicating the `RouterView` for the `SummaryView` into the different rule types. This worked out quite well for the `proxy` and `to` rules, but not for the `from` rules. Not sure what I am missing for them 😅 At least the route is not rerendered but the accordion closes when opening a policy summary from the `from` rules 🤔

Reply:
Ah sorry! Just noticed, the `v-if="route.child()"` needs to go on the `SummaryView`, so we always "render" the `RouterView` which is render-less, and conditionally render the `SummaryView` inside it if we have a route child.

I went back to 0ef4258ba25b35919005bf5171cb1bfd3aed67f9 and tried it there and it all seems fine.

I still don't completely get Vue's rendering algorithms sometimes 🤷‍♂️

Reply:
Perfect, that did the trick 👍 works like a charm now 😁